tools/uniprot_rest_interface/test-data/test2_retrieve.gff

```diff
@@ -1,16 +1,7 @@
 ##gff-version 3
 ##sequence-region S0DS17 1 369
-S0DS17	UniProtKB	Chain	1	369	.	.	.	ID=PRO_0000437163;Note=Cytochrome P450 momooxygenase apf8	
```

Comment:
Hmm... I guess these unrelated changes are because of rebasing against the master few days ago. Check out the commit history after that. Probably, the changes in PR #548 got caught and become  part of this PR. What is the best practice this moment?